test/ExecutionFramework.cpp

```diff
@@ -54,6 +56,32 @@ ExecutionFramework::ExecutionFramework() :
 	m_rpc.test_rewindToBlock(0);
 }
 
+std::pair<bool, string> ExecutionFramework::compareAndCreateMessage(
+	bytes const& _result,
+	bytes const& _expectation
+)
+{
+	if (_result == _expectation)
+		return std::make_pair(true, std::string{});
+	std::string message =
+			"Invalid encoded data\n"
+			"   Result                                                           Expectation\n";
+	auto resHex = boost::replace_all_copy(toHex(_result), "0", ".");
+	auto expHex = boost::replace_all_copy(toHex(_expectation), "0", ".");
+	for (size_t i = 0; i < std::max(resHex.size(), expHex.size()); i += 0x40)
+	{
+		std::string r{i >= resHex.size() ? string{} : resHex.substr(i, 0x40)};
+		std::string e{i > expHex.size() ? string{} : expHex.substr(i, 0x40)};
```

Comment:
Hm, actually I think we should change these to a bit longer variants (perhaps `res` and `exp`) for readability in the below section.

Reply:
Changed it to longer variants.